higlass/client.py

```diff
@@ -99,8 +102,8 @@ def __init__(
                 new_track["server"] = server
             else:
                 print(
```

Comment:
You either want to raise an `ValueError` exception or a [warning](https://pymotw.com/3/warnings/) here, rather than print to stderr.

Reply:
That's something for the next PR :) I only PEP8tified the existing print statement